factory/docker-compose.yml

```diff
@@ -0,0 +1,172 @@
+version: '3.8'
+
+# REPO_PREFIX is used in CI to deploy to other docker registries than dockerhub.
+
+services:
+
+  factory:
+    image: ${REPO_PREFIX-}cypress/factory
+    build:
+      dockerfile: factory.Dockerfile
+      target: factory
+      context: .
+      args:
+        BASE_IMAGE: ${BASE_IMAGE}
+        DEFAULT_NODE_VERSION: ${NODE_VERSION}
+      tags:
+       - ${REPO_PREFIX-}cypress/factory:${FACTORY_VERSION}
+    command: node -v
+
+  included:
+    image: ${REPO_PREFIX-}cypress/included
+    build:
+      target: included_image
+      context: .
+      args:
+        FACTORY_VERSION: ${FACTORY_VERSION}
+        CHROME_VERSION: ${CHROME_VERSION}
+        CYPRESS_VERSION: ${CYPRESS_VERSION}
+        EDGE_VERSION: ${EDGE_VERSION}
+        FIREFOX_VERSION: ${FIREFOX_VERSION}
+        YARN_VERSION: ${YARN_VERSION}
+        # WEBKIT_VERSION: ${WEBKIT_VERSION}
+      tags:
+       - ${REPO_PREFIX-}cypress/included:${INCLUDED_IMAGE_TAG}
+    command: node -v
+
+  browsers:
+    image: ${REPO_PREFIX-}cypress/browsers
+    build:
+      target: default_image
+      context: .
+      args:
+        FACTORY_VERSION: ${FACTORY_VERSION}
+        YARN_VERSION: ${YARN_VERSION}
+        CHROME_VERSION: ${CHROME_VERSION}
+        FIREFOX_VERSION: ${FIREFOX_VERSION}
+        EDGE_VERSION: ${EDGE_VERSION}
+      tags:
+       - ${REPO_PREFIX-}cypress/browsers:${BROWSERS_IMAGE_TAG}
+    command: node -v
+
+  base:
+    image: ${REPO_PREFIX-}cypress/base
+    build:
+      target: default_image
+      context: .
+      args:
+        FACTORY_VERSION: ${FACTORY_VERSION}
+        YARN_VERSION: ${YARN_VERSION}
+      tags:
+       - ${REPO_PREFIX-}cypress/base:${BASE_IMAGE_TAG}
+    command: node -v
+
+  ## Unused images that we could release in the future if we choose.
```

Comment:
I think it'd be better to remove the chrome/edge/firefox/webkit images for now till we decide we want to release these. the factory image can be used and this is dead code if checked in.

Reply:
I would prefer to keep these, they are useful to me when I want to develop/test specific combinations, such as only chrome.